content/devdocs/references/p2p_networking.md

```diff
@@ -315,7 +511,17 @@ message header has been omitted and the actual IP address has been
 replaced with a [RFC5737][] reserved IP address.)
 
 {{ <highlight text> }}
-GIVP message hex dump
+GivePeersMessage:
+```
+1a 00 00 00 00 00 00 00 03 00 00 00 5d 87 b2 76 
+70 17 9c 21 58 2f 70 17 94 67 29 79 70 17 ......... Full message
+------------------------------------------------------------------------
+0x0000 1a 00 00 00 00 00 00 00 ........................... Prefix
```

Comment:
This is not quite like it should be . First four bytes are the total size of the buffer that follows . The other four bytes are the message `Prefix` ID (e.g.one of `INTR`, `GETB`, ...) . Please fix that